WpNginxConfig.conf

```diff
@@ -0,0 +1,44 @@
+server {
+    listen 443 ssl;
+    listen [::]:443 ssl;
+    include snippets/self-signed.conf;
+    include snippets/ssl-params.conf;
+
+
+    server_name 192.168.50.2;
+    root /var/www/wordpress;
+
+    index index.php index.html index.htm;
+
+    location / {
+        #try_files $uri $uri/ =404;
+         try_files $uri $uri/ /index.php$is_args$args;
+    }
+
+    location ~ \.php$ {
+        include snippets/fastcgi-php.conf;
+        fastcgi_pass unix:/var/run/php/php7.4-fpm.sock;
+    }
+
+    location ~ /\.ht {
```

Comment:
what is it used for?

Reply:
line 20 = something about fastcgi module/ fastcgi server ?)

line 23 = location block that deals with .htacess files